WireUI/Sources/WireIndividualToTeamMigrationUI/Resources/Localizable.xcstrings

```diff
@@ -418,7 +418,7 @@
         "en" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "https://wire.com/en/pricing"
+            "value" : "https:\/\/wire.com\/en\/pricing"
```

Comment:
same here it should be within url.json

Reply:
It's actually moving to the main bundle.